stable/mlrun/templates/ui-deployment.yaml

```diff
@@ -33,6 +33,14 @@ spec:
           env:
           - name: MLRUN_API_PROXY_URL
             value: 'http://{{ include "mlrun.api.fullname" . }}:{{ .Values.api.service.port }}'
+          {{- if .Values.nuclioUIURL }}
+          - name: MLRUN_NUCLIO_UI_URL
+            value: {{ .Values.nuclioUIURL }}
+          {{- end }}
+          {{- if .Values.nuclioAPIURL }}
+          - name: MLRUN_NUCLIO_API_URL
+            value: {{ .Values.nuclioAPIURL }}
```

Comment:
It would be nice to add a comment on those 2 explaining the UI_URL i for displaying user-facing links, and the api url is for proxying on the nginx level

Reply:
Added